Assets/Scripts/Shooting.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Shooting : MonoBehaviour
+{
+
+    public float shootSpeed;
+    public float shootTimer;
+    
+    public Transform bulletSpawn1;
+    public Transform bulletSpawn2;
+    public Transform bulletSpawn3;
+    public Transform bulletSpawn4;
+    
+    public GameObject bullet;
+
+    private bool isShooting;
+
+    public bool autoShooting;
```

Comment:
```suggestion
    [SerializeField] private float shootSpeed;
    [SerializeField] private float shootTimer;
    
    [SerializeField] private Transform<> bulletSpawns;
    
    [SerializeField] private GameObject bullet;

    private bool _isShooting;

    [SerializeField] private bool autoShooting;
```

Meer protection en de 4 transforms kunnen  een list aan transforms worden.